platform/state/platform_state.proto

```diff
@@ -339,3 +339,17 @@ message NodeId {
    */
   uint64 id = 1;
 }
+
+/**
+ * A version of the platform software.
+ */
+message PlatformVersion {
+    /**
+     * Version of the HAPI specification that was used to serialize the state.
+     */
+    proto.SemanticVersion hapi_version = 1;
```

Comment:
Conceptually, platform should not have any relation to HAPI. Should this be `app_version` & `platform_version`?
Btw, I think we need to sync on the future of software version, because events also contain a version inside

Reply:
>Conceptually, platform should not have any relation to HAPI.

Currently, the platform has HAPI dependency and, unless there is a major architectural change (which very well may be), it will remain this way. 

> Should this be app_version & platform_version?

That would introduce more confusion. I believe there is no notion of `platform_version` at the moment. 

>Btw, I think we need to sync on the future of software version, because events also contain a version inside

Yes. I agree with this. I wonder though if it can be a part of release/0.54. Also, I talked to @jasperpotts about this, and he mentioned that in the Block Node we took similar approach:

```
message BlockHeader {
    /**
     * A version of the HAPI specification.<br/>
     * This is the API version that was used to serialize the block.
     */
    proto.SemanticVersion hapi_proto_version = 1;

    /**
     * A version of the consensus node software.<br/>
     * This is the software version that executed the transactions
     * within this block.
     */
    proto.SemanticVersion software_version = 2;
```